tools/test/test262/excludelist.orig.xml

```diff
@@ -3613,6 +3613,7 @@
     <test id="intl402/Locale/constructor-options-region-valid"><reason>TODO</reason></test>
     <test id="intl402/Locale/extensions-grandfathered"><reason>TODO</reason></test>
     <test id="intl402/Locale/likely-subtags-grandfathered"><reason>TODO</reason></test>
+    <test id="intl402/Locale/prototype/minimize/removing-likely-subtags-first-adds-likely-subtags"><reason>TODO</reason></test>
```

Comment:
What change cause this diff?

Reply:
This tc was originally failed, but accidentally removed in previous patch.
I'm not sure that how it passed CI
(this TC definitely fails on my 64bit environment)

Reply:
some TC's pass and fails depends on pc's icu on
you can remove it manually

Reply:
What is the version of icu installed in CI server now?
I'll match the version to that of CI

Reply:
The server uses icu 68.2